models/comment_thread.rb

```diff
@@ -73,6 +73,7 @@ class CommentThread < Content
   before_destroy :destroy_subscriptions
 
   scope :active_since, ->(from_time) { where(:last_activity_at => {:$gte => from_time}) }
+  scope :exclude_standalone, ->() { where(:context.ne => :standalone) }
```

Comment:
I'm not a fan of default scopes.  I've instead opted to create two scopes for specifying which context you would like.  The only time I think default scope makes sense is if you have some kind of soft delete flag on a model.  It is not expected functionality to return deleted objects in a query, so it makes sense there.